src/pkg/services/m365/api/events_test.go

```diff
@@ -212,3 +215,73 @@ func (suite *EventsAPIUnitSuite) TestBytesToEventable() {
 		})
 	}
 }
+
+type EventsAPIIntgSuite struct {
+	tester.Suite
+	credentials account.M365Config
+	ac          Client
+}
+
+func TestEventsAPIntgSuite(t *testing.T) {
+	suite.Run(t, &EventsAPIIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tester.M365AcctCredEnvs}),
+	})
+}
+
+func (suite *EventsAPIIntgSuite) SetupSuite() {
+	t := suite.T()
+
+	a := tester.NewM365Account(t)
+	m365, err := a.M365Config()
+	require.NoError(t, err, clues.ToCore(err))
+
+	suite.credentials = m365
+	suite.ac, err = NewClient(m365)
+	require.NoError(t, err, clues.ToCore(err))
+}
+
+func (suite *EventsAPIIntgSuite) TestRestoreLargeAttachment() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	userID := tester.M365UserID(suite.T())
+
+	folderName := testdata.DefaultRestoreConfig("eventrestoretest").Location
+	evts := suite.ac.Events()
+	calendar, err := evts.CreateContainer(ctx, userID, folderName, "")
+	require.NoError(t, err, clues.ToCore(err))
+
+	// Delete on exit
+	defer func() { _ = evts.DeleteContainer(ctx, userID, ptr.Val(calendar.GetId())) }()
+
```

Comment:
as long as the calendar follows the naming scheme for the purge script we can just remove this code